examples/seismic/acoustic/constant_example.py

```diff
@@ -24,66 +27,76 @@ def source(t, f0):
     return (1-2.*r**2)*np.exp(-r**2)
 
 
-def run(dimensions=(50, 50, 50), spacing=(20.0, 20.0, 20.0), tn=1000.0,
-        time_order=2, space_order=4, nbpml=40, dse='advanced', dle='advanced',
-        full_run=False):
+def setup(dimensions=(50, 50, 50), spacing=(15.0, 15.0, 15.0), tn=500.,
+          time_order=2, space_order=4, nbpml=10, vp=1.5, **kwargs):
+
     ndim = len(dimensions)
     origin = tuple([0.]*ndim)
     spacing = spacing[:ndim]
+    # Velocity model, two layers
+    true_vp = vp
+    # Source location
+    location = np.zeros((1, ndim), dtype=np.float32)
+    location[0, :-1] = [origin[i] + dimensions[i] * spacing[i] * .5
+                        for i in range(ndim-1)]
+    location[0, -1] = origin[-1] + 2 * spacing[-1]
+    # Receivers locations
+    receiver_coords = np.zeros((dimensions[0], ndim), dtype=np.float32)
+    receiver_coords[:, 0] = np.linspace(0, origin[0] +
+                                        (dimensions[0]-1) * spacing[0],
+                                        num=dimensions[0])
+    receiver_coords[:, 1:] = location[0, 1:]
 
-    # True velocity
-    true_vp = 2.
-
-    # Smooth velocity
-    initial_vp = 1.8
-
-    dm = 1. / (true_vp * true_vp) - 1. / (initial_vp * initial_vp)
-    model = Model(origin, spacing, dimensions, true_vp, nbpml=nbpml)
-    dm = np.ones(model.shape_domain, dtype=np.float32)*dm
+    # Define seismic data
+    model = Model(origin, spacing, dimensions, true_vp, nbpml=int(nbpml))
```

Comment:
`test_adjoint` adds `spc_order/2` to `nbpml` and with some pythons versions it's not an int anymore.